docs/resources/equinix_metal_connection.md

```diff
@@ -118,6 +121,7 @@ The following arguments are supported:
 * `facility` - (**Deprecated**) Facility where the connection will be created.   Use metro instead; read the [facility to metro migration guide](https://registry.terraform.io/providers/equinix/equinix/latest/docs/guides/migration_guide_facilities_to_metros_devices)
 * `redundancy` - (Required) Connection redundancy - redundant or primary.
 * `type` - (Required) Connection type - dedicated or shared.
+* `contact_email` - (Required) The preferred email used for communication and notifications about the Equinix Fabric interconnection. Required when using a Project API key. Optional and defaults to the primary user email address when using a User API key.
```

Comment:
Nit:  I'm not totally sold on labeling this `(Required)`, since it isn't always required; it looks like we went the opposite way with `project_id`, which is marked `(Optional)`.

Reply:
Fair enough. I'll merge and change that in a subsequent PR so as not to offend the pass status on tests.